config/formats/typescript-export-defition.ts

```diff
@@ -0,0 +1,145 @@
+
+import StyleDictionary from 'style-dictionary';
+import { format } from "prettier";
+const fs = require('fs');
+const path = require('path');
+
+const { fileHeader } = StyleDictionary.formatHelpers;
+
+/**
+ * unquoteTypes
+ * @description extract types from designTokenTypes (string with type definitions from file) and replace quoted types "Color" with unquoted Color
+ * @param output 
+ * @param designTokenTypes 
+ * @returns string with unquoted types
+ */
+const unquoteTypes = (output: string, designTokenTypes: string[]): string => {
+  // regex to match types like `type Color`
+  // const findTypesRegex = `type ([A-Za-z]+)`
+  // // extract types from designToken
+  // const types = [...designTokenTypes.matchAll(new RegExp(findTypesRegex, "g"))].map(match => match[1])
```

Comment:
Are these commented out code or annotations. Can we remove them?

Reply:
Yes.